.github/actions/skip/action.yml

```diff
@@ -0,0 +1,80 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+name: "Set Skip Env Var"
+description: "Action to set the SKIP_CI environment variable indicating that we should skip CI jobs"
+inputs:
+  paths:
+    description: >-
+      Set the SKIP_CI environment variable when and only when all the changed files located in one of the path,
+      the paths is shell-style pattern.
+    required: false
+    default: >-
+      "*.md"
+      "*.txt"
+      "skywalking-ui"
+      ".asf.yaml"
+      ".dlc.yaml"
+      ".licenserc.yaml"
+      "docs/menu.yml"
+      ".github/actions/skip/action.yml"
+      ".github/workflows/codeql.yaml"
+      "dist-material/release-docs"
+      "test/plugin/*"
+      "*/component-libraries.yml"
```

Comment:
You should update this. I think these are legecy.

Reply:
Removed